### DISCUSSION TITLE
Welcome

### DISCUSSION THREAD

**Discussion by kannagi0303** [kannagi0303/yt-dlp-gui#12] [General]
Welcome everyone

**Comment by BBChapnick** [comment]
Hi kannagi,
             I just wanted to say EXCELLENT WORK. I struggled with slow speeds with youtube-dl and quickly switched to yt-dlp and luckily I found this great GUI. Instructions were clear put ffmpeg.exe and  yt-dlp_x86.exe into bin folder and found it worked PERFECTLY as soon as I renamed the yt-dlp_x86.exe to: yt-dlp.exe   VERY HAPPY WITH THIS!

**Comment by ghost** [comment]
hi, great program, thank you for making it. this makes the process way easier and more user-friendly. there are other gui's but they suck compared to this one. I'll share it so that other people can use it as well
